.build/check_license.sh

```diff
@@ -9,7 +9,7 @@ run_uber_licence() {
   # Ok, somebody hasn't run `npm install -g uber-licence`, that's cool
   if ! which "$bin" >/dev/null ; then
     npm install "$bin" >/dev/null
-    bin="./node_modules/uber-licence/bin/licence"
```

Comment:
this still won't fully work, as `which uber-licence` does not puck up the local modules and will run the install every time.

```bash
$ which uber-licence
$ find . -name uber-licence
./node_modules/.bin/uber-licence
```

The proper way to fix would be check `find uber-licence` or `node_modules/uber-licence`, else install.